src/Groups/perm.jl

```diff
@@ -149,6 +149,18 @@ function perm(L::AbstractVector{<:IntegerUnion})
   return PermGroupElem(symmetric_group(length(L)), GAPWrap.PermList(GapObj(L;recursive=true)))
 end
 
+"""
+    smaller_degree_permutation_representation(G::PermGroup) -> PermGroup, map
+  
+Return an isomorphic permutation group of smaller degree and the isomorphism.
```

Comment:
We cannot guarantee a strictly smaller degree. It may happen that the returned isomorphism is the identity map.